### ISSUE TITLE
"Getting Started" Notebook is not up to date

### ISSUE THREAD

**Issue by Anbue63** [PIK-ICoN/NetworkDynamics.jl#76]
Hey,

The "getting_started_with_network_dynamics.ipynb" is not up to date anymore when using NetworkDynamics 0.5.0.
Probably since the edge definition, removing source and destination, was changed?
I changed the definition for the diffusionvertex! function to the version which is given in the Docs:
```
function diffusionvertex!(dv, v, edges, p, t)
    # usually v, edges are arrays, hence we use the broadcasting operator .
    dv .= 0.
    for e in edges
        dv .+= e
    end
    nothing
end
```
This fixed it for me! (I would have liked to make a pull request but jupyter notebooks do not run on my machine.)

**Comment by lindnemi** [comment]
Hi @Anbue63 , thanks for letting us know! @fdrauschk @acejolanda could you look into that?

**Comment by acejolanda** [comment]
sure!

**Comment by lindnemi** [comment]
fixed f6f08138dbdd10005719718b012b2e2065309dcc
